### PR TITLE
This adds the LoCoBot robot to the robotics.ros site. This robot is a…

### PR DESCRIPTION
**Pull request by swinterbotix**
…ssembled by Trossen Robotics and includes the installation of PyRobot on its computer.

### CODE REVIEW

File: _posts/2020-01-23-locobot.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- 'indoor'
+- 'manipulator'
+- 'mobile base'
+- 'mobile manipulator'
+- 'pyrobot'
+- 'research'
+- 'wheeled'
+twitter_text: Checkout the LoCoBot
+website: https://www.trossenrobotics.com/locobot-pyrobot-ros-rover.aspx
+wiki_homepage: http://wiki.ros.org/locobot
```
Comment (tfoote) [inline _posts/2020-01-23-locobot.md:24]: This link does not appear to exist.

File: _posts/2020-01-23-locobot.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+TODO Fill in the long description/article here.
```
Comment (tfoote) [inline _posts/2020-01-23-locobot.md]: There's still a TODO here.

File: _posts/2020-01-23-locobot.md (hunk 1 above)
Comment (swinterbotix) [reply _posts/2020-01-23-locobot.md:24]: The link now exists

File: _posts/2020-01-23-locobot.md (hunk 2 above)
Comment (swinterbotix) [reply _posts/2020-01-23-locobot.md]: A long description was added.

**Comment by swinterbotix** [comment]
Hi @tfoote,

I pushed a commit with the changes you asked for.

**Pull request merged**